neqo-http3/src/features/extended_connect/tests/webtransport/sessions.rs

```diff
@@ -154,6 +157,46 @@ fn wt_session_respone_with_1xx() {
     assert_eq!(wt_session_id, wt_server_session.stream_id());
 }
 
+#[test]
+fn wt_session_response_with_redirect() {
+    let mut wt = WtTest::new();
+
+    let wt_session_id = wt
+        .client
+        .webtransport_create_session(now(), &("https", "something.com", "/"), &[])
+        .unwrap();
+    wt.exchange_packets();
+
+    let mut wt_server_session = None;
+    while let Some(event) = wt.server.next_event() {
+        if let Http3ServerEvent::WebTransport(WebTransportServerEvent::NewSession {
+            session,
+            headers,
+        }) = event
+        {
+            assert!(
+                headers
+                    .iter()
+                    .any(|h| h.name() == ":method" && h.value() == "CONNECT")
+                    && headers
+                        .iter()
+                        .any(|h| h.name() == ":protocol" && h.value() == "webtransport")
+            );
+            wt_server_session = Some(session);
+        }
+    }
+
+    let mut wt_server_session = wt_server_session.unwrap();
+
+    let accept_res: WebTransportSessionAcceptAction = WebTransportSessionAcceptAction::Redirect(
+        [Header::new(":status", "302"), Header::new("location", "/")].to_vec(),
+    );
+    wt_server_session.response(&accept_res).unwrap();
```

Comment:
Why not reuse the current function `negotiate_wt_session` to send response?
It should be possible to do:
```
 let (wt_session_id, _wt_session) =
    wt.negotiate_wt_session(&accept_res);
```

Reply:
Done